Sources/Contour/Transport/RabbitMQ/BusConsumerConfigurationEx.cs

```diff
@@ -38,6 +40,17 @@ public static IReceiverConfigurator WithQoS(this IReceiverConfigurator builder,
         }
 
         /// <summary>
+        /// Returns current QoS settings
+        /// </summary>
+        /// <param name="builder"></param>
+        /// <returns></returns>
+        public static Maybe<QoSParams> GetQoS(this IReceiverConfigurator builder)
+        {
+            var qos = ((RabbitReceiverOptions)((ReceiverConfiguration)builder).Options).GetQoS();
```

Comment:
Maybe<T> pattern is used to escape needless exceptions. Check type safety and use Maybe<T> to return value.